apps/docs-app/docs/features/deployment/providers.md

```diff
@@ -426,3 +426,152 @@ jobs:
           echo "DRY_RUN_OPTION=$DRY_RUN_OPTION"
           npx angular-cli-ghpages --no-silent --dir="${{env.TARGET_DIR}}" $CNAME_OPTION $DRY_RUN_OPTION
 ```
+
+## Zerops
+
+Analog supports deploying on [Zerops](https://zerops.io) with some additional configuration which provides you with more control over your analog projects.
+
+:::info
+
+### 🚀 Analog x Zerops Quickrun
+
+Want to test running Analog on Zerops without installing or setting up anything? Using repositories [**Zerops x Analog - Static**](https://github.com/zeropsio/recipe-analog-static) or [**Zerops x Analog - SSR**](https://github.com/zeropsio/recipe-analog-nodejs) on Node.js you can deploy example Analog site with a single click.
+
+#### Deploy with a single click
+
+- [**Analog Static & Analog SSR**](https://app.zerops.io/recipe/analog) (Test both at once)
+- [**Analog Static**](http://app.zerops.io/recipe/analog-static)
+- [**Analog SSR**](http://app.zerops.io/recipe/analog-nodejs)
+  :::
```

Comment:
Please remove this block. It sounds like marketing and not docs

```suggestion
```

Reply:
Sorry @brandonroberts, not intentially marketing-ish, @nermalcat69 just took the base guide from [Astro's](https://docs.astro.build/en/guides/deploy/zerops/) docs where it was more or less in line with other guides. :) 

We'll tone it down to match style of other guides in Analog.